bom-latest/pom.xml

```diff
@@ -171,6 +184,11 @@
                 <artifactId>git-server</artifactId>
                 <version>1.9</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>jackson2-api</artifactId>
+                <version>2.10.2</version>
```

Comment:
shouldn't this be:
```suggestion
                <version>2.11.0</version>
```

Reply:
Can be. Also I just found that PMD uses 2.9.8, so maybe it needs to be pulled forward too

Reply:
@chriskilding It doesn't care.

Reply:
Anyway if we are introducing plugins to the BOM, may as well pick the newest version now, or Dependabot will offer it soon enough.

Reply:
Looks like it wasn't quite that simple... had to backpedal that for now.

- pipeline-model-api 1.7.0 wants Jackson 2.9.8
- CasC test-harness 1.36 wants Jackson 2.10.2 (though I don't get how we are even getting 1.36, as the BOM declares 1.41 and nothing seems to override this)

Reply:
> I don't get how we are even getting 1.36, as the BOM declares 1.41

In all LTS lines?

Reply:
Not sure I understand the problem. One plugin wants 2.9.8+, another wants 2.10.2+, we pick 2.11.0, all should be fine?

Reply:
> > I don't get how we are even getting 1.36, as the BOM declares 1.41
> 
> In all LTS lines?

Ah, 2.204.x was the culprit.

Reply:
The sample POM doesn't directly depend on Jackson at the moment, so I guess that left the version conflict unresolved. I'll add a dependency.

Reply:
pipeline-model-api seems happy to be bumped to Jackson 2.11.0 though. https://github.com/jenkinsci/pipeline-model-definition-plugin/pull/399